chart/open-feature-operator/README.md

```diff
@@ -92,6 +92,7 @@ The command removes all the Kubernetes components associated with the chart and
 
 ### Sidecar configuration
 
+<!-- x-release-please-start-version -->
```

Comment:
😢 sorry, this placement caused helm readme validation to fail

Could you please move this start tag to above `## Configuration` at line  85

```
<!-- x-release-please-start-version -->
## Configuration
```

Validated in my setup and this solves the issue